application/common/tests/test_dos_db_connection.py

```diff
@@ -22,89 +22,78 @@
 
 @patch(f"{FILE_PATH}.connection_to_db")
 @patch(f"{FILE_PATH}.get_secret")
-def test_connect_to_dos_db_replica(mock_get_secret, mock_connection_to_db):
+def test_connect_to_db_reader(mock_get_secret, mock_connection_to_db):
     # Arrange
-    mock_get_secret.return_value = {"DB_REPLICA_SECRET_KEY": DB_PASSWORD}
-    environ["DB_REPLICA_SECRET_NAME"] = "my_secret_name"
-    environ["DB_REPLICA_SERVER"] = DB_SERVER
+    mock_get_secret.return_value = {"DB_READER_SECRET_KEY": DB_PASSWORD}
+    environ["DB_READER_SECRET_NAME"] = "my_secret_name"
+    environ["DB_READER_SERVER"] = DB_WRITER_SERVER
```

Comment:
This is it correct the `DB_READER_SERVER` env being set with the `DB_WRITER_SERVER` ?